src/lib/game/world/map.js

```diff
@@ -86,6 +92,10 @@ class Map extends THREE.Group {
           }
 
           this.add(m2);
+
+          if (this.world !== null && m2.billboards.length > 0) {
+            this.world.addBillboardedM2(m2);
+          }
```

Comment:
My only thought there is that sometimes the world consists of more than one map (phasing does this). I suppose that's just a matter of calling ```update``` on all maps currently loaded into the world.